src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -264,10 +270,73 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
             return 0;
         }
 
+        private static bool All<TNode>(SyntaxList<TNode> items, Func<TNode, bool> predicate)
+            where TNode : SyntaxNode
+        {
+            bool result = true;
+            for (int i = 0; i < items.Count; i++)
+            {
+                result &= predicate(items[i]);
+            }
+
+            return result;
+        }
+
+        private static bool IsAutoAccessorList(SyntaxToken token)
+            => token.Parent is AccessorListSyntax accessorList &&
+               All(accessorList.Accessors, a => a.Body == null);
+
+        private static bool IsAutoAccessorList(SyntaxNode? node)
+        {
+            if (node?.Parent is AccessorListSyntax accessorList)
+            {
+                return All(accessorList.Accessors, a => a.Body == null);
+            }
+
+            return false;
+        }
+
+        private static bool IsAccessorListFollowedByInitializer(SyntaxToken token) =>
+            token.Parent is AccessorListSyntax accessorList &&
+            accessorList.Parent is PropertyDeclarationSyntax propertyDeclarationSyntax &&
+                    propertyDeclarationSyntax.Initializer != null;
```

Comment:
1. this indentation is weird.
2. you aren't handling indexers afaict.  so this means we have a test hole for the equivalent case here for indexers.

Reply:
I am going to commit more tests for indexers and props, but consider that indexers can't have initializers, therefore many cases are not syntactically correct.
This is the reason why the `BasePropertyDeclarationSyntax` does not have the `Initializer` property.

Reply:
Basically the situation is the following:
- The indexers with the correct syntax are okay (new tests are coming)
- The indexers with incorrect syntax looks weird but they are **identical** to what you obtained before my PR

For this reason I am not going to add the tests for the indexers with the initializers as the situation did not change with my PR.